nvidia_triton/metadata.csv

```diff
@@ -0,0 +1,43 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+nvidia_triton.nv.cache.insertion.duration,gauge,,,,"Total cache insertion duration, in microseconds",0,nvidia_triton,,
```

Comment:
you have a column `unit_name` where you could specify the unit (microseconds here). List of metrics [here](https://github.com/DataDog/integrations-core/blob/c45614423f51f2c8337b99140359de0eef6f98f4/ddev/src/ddev/cli/validate/metadata_utils.py#L32). This way the unit will be displayed in the widget automatically